airflow/serialization/serialized_objects.py

```diff
@@ -118,7 +118,7 @@ def encode_timezone(var: Timezone) -> Union[str, int]:
 
 def decode_timezone(var: Union[str, int]) -> Timezone:
     """Decode a previously serialized Pendulum Timezone."""
-    return pendulum.timezone(var)
+    return pendulum.tz.timezone(var)
```

Comment:
What is the error here? `timezone` should be exposed as a top-level function.

Reply:
I know it's exposed in the __init__.py but it is saying that pendulum is not callable for some reason. This solved that issue and I checked across airflow project it's used in a both ways.
Actual error:
```
airflow/serialization/serialized_objects.py:121: error: Module not callable
        return pendulum.timezone(var)
               ^
```

Reply:
Actually I think it’s complaining that `pendulum.timezone` is not callable (because there is actually a `pendulum.timezone` package and that got Mypy confused). Pendulum’s package structure is really weird :(